index.html

```diff
@@ -632,7 +632,7 @@
 need of protection.
 
 <dfn data-lt="governance">Data governance</dfn> is the system of principles that regulate [=information flows=].
-[=Data governance=] determines how
+[=Data governance=] determines
 which [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
```

Comment:
The "how" here was actually doing work, but I agree with @chaals that it was confusing. The original meaning was that "Data governance determines which actors can collect data, **how those actors can collect the data**, and how they're allowed to process it." This change loses that second piece. Do we want to keep it?